examples/structured_data/time_series_anomaly_using_autoencoder.py

```diff
@@ -0,0 +1,344 @@
+"""
+Title: Timeseries anomaly detection using an Autoencoder
+Author: [pavithrasv](https://github.com/pavithrasv)
+Date created: 2020/05/31
+Last modified: 2020/05/31
+Description: Detect anomalies in a timeseries using an Autoencoder.
+"""
+
+"""
+## Introduction
+
+This script demonstrates how you can use a reconstruction convolutional autoencoder model
+to detect anomolies in timeseries data.
+
+We train the model on timeseries data which does not have any anomalies. We will use the
+[Numenta Anomaly Benchmark](https://www.kaggle.com/boltzmannbrain/nab?) dataset to
```

Comment:
No `?` in url; please describe the dataset used to provide context: what does the data represent? why is this a good fit for demonstrating anomaly detection?

Reply:
Removed this text form here and added more explanation in the load data section below.